backend/tests/test_author_api.py

```diff
@@ -43,3 +66,56 @@ def test_get_friends(self, client, test_user, friend_user):
         assert response.data["Author"] is not None
         assert response.data["Author"][0] == friend_user[0].get_full_user_id()
         assert response.data["Author"][1] == friend_user[1].get_full_user_id()
+
+        '''
+            Checking scenario where the user does not exist, 404 Error
+        '''
```

Comment:
Can we get rid of all these block comments. They should only be used as a doc string and not as a regular comment. Just convert it to a regular comment.

Reply:
There are a few in this file and the other PR that got merged as well.

Reply:
Sure.

Reply:
Done!